docs/myria-python/index.md

```diff
@@ -178,6 +178,89 @@ query = MyriaQuery.parallel_import(relation=relation, work=work)
 print query.status
 ```
 
+### Part 4. Python Expressions
+
+Myria allows arbitrary Python Functions as expressions which can be used in transformations like map, flatmap and aggregates.
+#### Registering a Python Function
+Python function needs to be registered before it can be used. Myria-python can be used to register python functions, list registered python functions as well as get details of the registered function.
+```python
+from myria.udf import MyriaPythonFunction
+from raco.types import LONG_TYPE
+
+#create connection
+connection = MyriaConnection(rest_url='http://localhost:8753',execution_url='http://localhost:8080')
+
+#register Python functions
+outType= "LONG_TYPE"
+def pyIsPrime(dt):
+    import math
+    n = dt[0][0]
+    if n % 2 == 0 and n > 2: 
+        return False
+    for i in range(3, int(math.sqrt(n)) + 1, 2):
+        if n % i == 0:
+            return 0
+    return 1
+
+MyriaPythonFunction(pyIsPrime, outType).register()
```

Comment:
I just tried it out and was able to use the @myria_function decorator, but it should be noted that I'm on a slightly old version of Myria at this point. The following code worked:
```
from myria import *
%load_ext myria
connection = MyriaConnection(rest_url='http://localhost:8753')
%connect http://localhost:8753
from myria.udf import myria_function

@myria_function(name='bar', output_type="LONG_TYPE", connection=connection)
def bar(dt):
    import math
    n = dt[0][0]
    if n % 2 == 0 and n > 2: 
        return False
    for i in range(3, int(math.sqrt(n)) + 1, 2):
        if n % i == 0:
            return 0
    return 1
```